### ISSUE TITLE
Elytrian Balance Fix

### ISSUE THREAD

**Issue by yeevee** [apace100/origins-fabric#130]
The damage increase from flying elytrians is able to one-hit KO a player with full Protection IV Netherite Armor with Elytra wings. The damage increase should be lowered to make this more balanced.

**Comment by CyroxTM** [comment]
Not sure if  suggestions are taken on matters like these but I'd say in my opinion the damage should scale with the player speed (velocity) but it would have a starting cap and a max cap for example:
-Bonus damage starts ramping up only after reaching speeds similar to speed 4 (number is just for example purposes) sprinting speed while gliding (to prevent quickly double tapping space to get extra damage) and it caps at 75% (reason same as before) max extra damage
Just in my opinion

**Comment by yeevee** [comment]
> Not sure if suggestions are taken on matters like these but I'd say in my opinion the damage should scale with the player speed (velocity) but it would have a starting cap and a max cap for example:
> -Bonus damage starts ramping up only after reaching speeds similar to speed 4 (number is just for example purposes) sprinting speed while gliding (to prevent quickly double tapping space to get extra damage) and it caps at 75% (reason same as before) max extra damage
> Just in my opinion

Yes I 100% agree with this compromise. Having immediate double damage increase for sudden flight time is quite overpowered atm.
